.eslintrc.json

```diff
@@ -70,9 +71,9 @@
       "space-before-function-paren": ["error", "never"],
       "space-infix-ops": ["error"],
       "spaced-comment": ["error", "always"],
-      "strict": ["error", "global"],
+      "strict": ["off"],
```

Comment:
Why are we turning this off?
I referred to [Eslint - Rules - Strict](https://eslint.org/docs/rules/strict). I'm a little confused because it says:
> In ECMAScript modules, which always have strict mode semantics, the directives are unnecessary.
Further down it says:
> In a codebase that has both strict and non-strict code, either turn this rule off, or selectively disable it where necessary.

I thought it was selectively turned on by adding `"use strict";` at the top of the file.

Reply:
I specified `{ "parserOptions": { "sourceType": "module" } }` which reflects reality because we're using `import React from 'react'` style imports with webpack (ECMASCript modules). Once you do that, ESLint raises an error for each `"use strict"` declaration, because they are actually not required.